run_single_test.py

```diff
@@ -39,11 +40,13 @@ def main() -> None:
     parser.add_argument('--cross-file', action='store', help='File describing cross compilation environment.')
     parser.add_argument('--native-file', action='store', help='File describing native compilation environment.')
     parser.add_argument('--use-tmpdir', action='store_true', help='Use tmp directory for temporary files.')
+    parser.add_argument('--quick', action='store_true', help='Skip some compiler and tool checking')
```

Comment:
I usually use `./run_project_tests.py --only common/1' '` or similar here, because I keep forgetting that run_single_test correctly handles failing/ now. Maybe I should switch over...

Reply:
Cherry-picked to master.